.vib/schema-registry/cypress/cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8081",
```

Comment:
We leave this as `http://localhost` by default, as the value does not really matter here. The VIB Cypress action replaces the value with the URL from the deployment LB. It doesn't really add value to have it with a different port number (even if the chart uses `!80` by default.